docker/Dockerfile.dev

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7
```

Comment:
Be good to pin the version. And use Alpine?

Reply:
updated and now pinned to python 3.7.2 using alpine linux 3.8